### PR TITLE
Minor fixes on code

### PR DESCRIPTION
**Pull request by csambrosio01**

### CODE REVIEW

File: src/scheduler/Scheduler.java [hunk 1]
```diff
@@ -72,7 +72,7 @@ private void executeProcess(PCB processPCB) {
                     processPCB.setWaitTo2();
                     ProcessList.addBlockedProcess(processPCB);
                 } else {
```
Comment (caio-k) [inline src/scheduler/Scheduler.java]: The round robin algorithm does not need lines 66 and 67, and line 79 should be replaced by `ProcessList.addReadyProcessInLastPosition (processPCB);` for the round robin algorithm. Anything look like I did in the createRoundRobin branch

File: src/scheduler/Scheduler.java [hunk 2]
```diff
@@ -109,7 +111,7 @@ public void run() {
             }
         }
 
-        logger.addMessage("AVERAGE_EXCHANGES", changes);
+        logger.addMessage("AVERAGE_EXCHANGES", changes / numOfProcess);
```
Comment (csambrosio01) [inline src/scheduler/Scheduler.java]: If you look at file gave by professor you could see that the correct number should be only `changes` (as it was), so this division is wrong

File: src/scheduler/Scheduler.java [hunk 3]
```diff
@@ -69,12 +68,15 @@ private void executeProcess(PCB processPCB, boolean roundRobin) {
...
                     processPCB.setWaitTo2();
+                    ProcessList.decreaseBlockedListWait(roundRobin);
                     ProcessList.addBlockedProcess(processPCB);
                 } else {
                     if (roundRobin) {
                         process.setState(State.READY);
                         ProcessList.addReadyProcessInLastPosition(processPCB);
+                        ProcessList.decreaseBlockedListWait(roundRobin);
                     } else {
+                        ProcessList.decreaseBlockedListWait(roundRobin);
```
Comment (csambrosio01) [inline src/scheduler/Scheduler.java]: By your logic, if a process ends, blocked list will not decrease wait of blocked process. This should be moved to last line before end of `do...while()`.

File: src/scheduler/ProcessList.java [hunk 4]
```diff
@@ -34,17 +34,21 @@ public static PCB removeNextInReadyList() {
...
 				PCB pcb = blockedList.remove(i);
 				pcb.decreaseWait();
 				if (pcb.getWait() == 0) {
 					pcb.getProcess().setState(State.READY);
-					if (pcb.getCredit() > 0 || !roundRobin) {
+					if (pcb.getCredit() > 0) {
 						addReadyProcess(pcb);
 					} else {
-						addReadyProcessInLastPosition(pcb);
+						if (allProcessInReadyListWithZEROCredit()) {
```
Comment (csambrosio01) [inline src/scheduler/ProcessList.java]: You are spending time calculating this boolean, when it does same thing as before. You should keep variable `roundRobin`. Also I think this logic doesn't make sense. You made a refact on if, but it does same thing as before. Better to keep previous if.

**Pull request merged**